features/docker.feature

```diff
@@ -0,0 +1,285 @@
+Feature: Command output from box command
+  service-manager should return the correct output from box commands
+
+  Scenario Outline: Boot and execute box commands
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    begin
+      require 'vagrant-libvirt'
+    rescue LoadError
+      # NOOP
+    end
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I run `bundle exec vagrant up --provider <provider>`
+    Then the exit status should be 0
+
+    ####################################################################################################################
+    # BOX command
+    ####################################################################################################################
+    When I run `bundle exec vagrant service-manager box`
+    Then the exit status should be 1
+    And stdout from "bundle exec vagrant service-manager box" should contain:
+    """
+    Usage: vagrant service-manager box <sub-command> [options]
+
+    Sub-Command:
+          version    display version and release information about the running VM
+          ip         display routable IP address of the running VM
+
+    Options:
+          --script-readable  display information in a script readable format
+          -h, --help         print this help
+    """
+
+    When I successfully run `bundle exec vagrant service-manager box --help`
+    Then the exit status should be 0
+    And stdout from "bundle exec vagrant service-manager box --help" should contain:
+    """
+    Usage: vagrant service-manager box <sub-command> [options]
+
+    Sub-Command:
+          version    display version and release information about the running VM
+          ip         display routable IP address of the running VM
+
+    Options:
+          --script-readable  display information in a script readable format
+          -h, --help         print this help
+
+    Examples:
+          vagrant service-manager box version
+          vagrant service-manager box version --script-readable
+          vagrant service-manager box ip
+          vagrant service-manager box ip --script-readable
+    """
+
+    When I successfully run `bundle exec vagrant service-manager box ip`
```

Comment:
AFAICT 'box version' was not tested in box.feature either. I just combined the existing tests. I did not add new ones.